### PR TITLE
in app rating through StoreKit

### PR DESCRIPTION
**Pull request by ShresthPratapSingh**
[Feature] : Implemented in app rating using storekit. 

- On "Rate Us" tap an app store alert pops up, in which the user can tap relevant stars

- Users can have seemless rating experience without leaving the app.

- earlier the user was redirected to the apppstore every time on tapping Rate Us.

- SKStoreReviewController is available for iOS 10.3 and above. so for users below ios 10.3(if any) they are redirected to the app store only for ratings.

Feature screenshot

### CODE REVIEW

**Comment by anubhavsingh16** [comment]
@ShresthPratapSingh I already send PR for app rating.

**Comment by anubhavsingh16** [comment]
So can you close your Pull Request?

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -805,14 +788,14 @@
...
-				DEVELOPMENT_TEAM = 325LAYFA25;
+				DEVELOPMENT_TEAM = Y8HPPY9DLS;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = AmahiAnywhere/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywher;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:798]: This introduces a bad bundle id

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -783,14 +766,14 @@
...
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:776]: This introduces a bad bundle id

File: AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -783,14 +766,14 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 325LAYFA25;
+				DEVELOPMENT_TEAM = Y8HPPY9DLS;
```
Comment (cpg) [inline AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj:769]: This changes the team and should not be part of the commit

**Comment by ShresthPratapSingh** [comment]
@cpg  Thank you for your suggestions. I've noted the changes and will update the PR.

**Comment by anubhavsingh16** [comment]
#295 is updated one of this PR.
